pwiz_tools/Skyline/Model/ModifiedSequence.cs

```diff
@@ -141,6 +141,61 @@ public ModifiedSequence(string unmodifiedSequence, IEnumerable<Modification> exp
             _defaultMassType = defaultMassType;
         }
 
+        public ModifiedSequence(string modifiedSequence, MassType defaultMassType)

```

Comment:
It looks like this only gets used by test code.
Could this functionality of constructing a ModifiedSequence from a string be moved into the TestUtil project?

Reply:
I suppose it could, but it's pretty useful functionality and I was surprised it wasn't already in the Skyline code. Most of the code was copied from ModificiationMatcher, even.

Reply:
Is there any way to share code with ModificationMatcher, then? Or at least add a comment, in case one gets changed/improved.

Reply:
I'll see about factoring out the mutual enumeration of modification blocks in the string.